src/authorization/repository/entity.repository.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import { UpdateEntityInput } from 'src/schema/graphql.schema';
+import { DataSource } from 'typeorm';
+import EntityModel from '../entity/entity.entity';
+import { BaseRepository } from './base.repository';
+
+@Injectable()
+export class EntityModelRepository extends BaseRepository<EntityModel> {
+  constructor(private dataSource: DataSource) {
+    super(EntityModel, dataSource);
+  }
+
+  async getAllEntities(): Promise<EntityModel[]> {
+    return this.find();
+  }
+
+  async getEntityById(id: string) {
+    return this.findOneBy({ id });
+  }
+
+  async updateEntityById(
+    id: string,
+    role: UpdateEntityInput,
+  ): Promise<Boolean> {
```

Comment:
Should use boolean instead of Boolean as the return type. See https://github.com/typescript-eslint/typescript-eslint/blob/v4.19.0/packages/eslint-plugin/docs/rules/ban-types.md#examples